comid/measurement.go

```diff
@@ -373,9 +375,9 @@ func NewPSAMeasurement(psaRefValID PSARefValID) *Measurement {
 
 // NewCCAMeasurement instantiates a new measurement-map with the key set to the
 // supplied CCA refval-id
-func NewCCAMeasurement(ccaRefValID CCARefValID) *Measurement {
+func NewCCAMeasurement(ccaPlatformConfigID CCAPlatformConfigID) *Measurement {
```

Comment:
`NewCCAPlatformConfigIDMeasurement`?

Reply:
That's too long

Reply:
maybe, but it's consistent with the naming convention.
`NewCCAMeasurement` OTOH is not really meaningful.